helper-cli/src/main/kotlin/commands/ListLicensesCommand.kt

```diff
@@ -213,3 +230,10 @@ private fun TextLocation.resolve(baseDir: File): String? {
 
     return lines.subList(startLine - 1, endLine).joinToString(separator = "\n")
 }
+
+private fun Provenance.writeValueAsString(): String =
```

Comment:
I probably just need some background information (in the form of a code comment for this function) what's the intended benefit of the function instead of using `Provenance.toString()`, or at `RemoteArtifact.toStrong()` and `VcsInfo.toString()`. Do you want / require a more compact string representation? If so, why? For a better human-readable overview, omitting fields / data that is irrelevant in this context?

Reply:
Done.

Reply:
Yes, thanks.